langchain/src/vectorstores/vectara.ts

```diff
@@ -163,6 +168,61 @@ export class VectaraStore extends VectorStore {
     }
   }
 
+  async addFiles(
```

Comment:
Can this just use the text loader?

https://js.langchain.com/docs/modules/data_connection/document_loaders/integrations/file_loaders/text

Reply:
The purpose of the addFiles function is to upload and index any document not just text. Vectara accepts files in various formats like PDF, PPT, etc. Those files are then processed to extract and chunk the text. This simplifies the steps for developers. 

The Vectara API call is: https://docs.vectara.com/docs/api-reference/indexing-apis/file-upload/file-upload

The python version of this function can be found here: https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/vectorstores/vectara.py#L134

cc @ofermend

Reply:
Gotcha, ok